modules/task/README.md

```diff
@@ -38,10 +38,8 @@ func newBackend(host service.Host) (task.Backend, error) {
 
 func runActivity(input string) error {
   // do things
-  results := "results"
-  if err := task.Enqueue(updateCache, input, results); err != nil {
-    return err
-  }
+  results := // calculate results here
```

Comment:
I prefer it to be compilable: 
It makes easy to test examples and for us to think hard about APIs

Minor thing anyway.